package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "build": "tsc",
     "pretest": "npm run build",
-    "test": "mocha",
+    "test": "mocha -t 5000",
```

Comment:
Please move the timeout config to https://github.com/strongloop/loopback-next-hello-world/blob/master/test/mocha.opts

Do we really need 5s for every test? Can we increase the timeout only in the slow test?

```ts
it('Throws error for not starting', function () {
  this.timeout(5000);
  // etc.
}
```

(It's important to use a regular function, not an arrow function, so that `this` points to Mocha's context.)